aptos-move/framework/aptos-framework/sources/permissioned_signer.move

```diff
@@ -81,7 +81,7 @@ module aptos_framework::permissioned_signer {
     ///
     /// This handle is storable and thus should be treated very carefully as it serves similar functionality
     /// as signer delegation.
-    enum StorablePermissionedHandle has store {
+    enum StorablePermissionedHandle has copy, drop, store {
```

Comment:
We need to avoid changing ability here: each handle is tied to some storage information. Making them copy/drop will cause either garbage data in the DB and can break some invariants.